crates/subspace-node/src/domain/cli.rs

```diff
@@ -63,6 +64,10 @@ pub struct DomainCli {
     #[clap(long)]
     pub relayer_id: Option<String>,
 
+    /// Run the node as an Operator
+    #[arg(long, conflicts_with = "validator")]
```

Comment:
Making it conflict with `--validator` and `--validator` do nothing is confusing. I have created https://github.com/subspace/subspace/issues/1727 to fix this kind of stuff on consensus side at least, but we can also include domains there as well. Until then I think `--validator` should have the same behavior as `--operator` and vice versa.